babel.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  presets: ["next/babel"],
```

Comment:
Does this have any effect? The Next.js docs over [here](https://nextjs.org/docs/advanced-features/customizing-babel-config) would suggest that `next/babel` is already included 🤔